rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -476,15 +553,17 @@ def _get_staking_events(
         if new_addresses:
             new_events = self._get_new_staking_events_graph(
                 addresses=addresses,
+                identity_address_map=identity_address_map,
                 from_timestamp=Timestamp(0),
                 to_timestamp=to_timestamp,
             )
             all_new_events.extend(new_events)
 
         # Request existing DB addresses' events
-        if existing_addresses and min_from_timestamp <= to_timestamp:
+        if existing_addresses and to_timestamp > min_from_timestamp:
```

Comment:
you may have conflicts here. I think we need to merge bugfixes to develop and you rebased on top or this will conflict.